### ISSUE TITLE
[Feature Request] GuardDuty managed log source

### ISSUE THREAD

**Issue by britton-from-notion** [matanolabs/matano#191]
## TL;DR

Managed AWS GuardDuty log source support

## Feature Request

Hey there! Love the project, thanks for all your work on it.

The Matano documentation lists a few [Supported managed AWS log sources](https://www.matano.dev/docs/log-sources/managed/aws), however it doesn't appear that [GuardDuty](https://docs.aws.amazon.com/guardduty/latest/ug/guardduty_findings.html) is currently supported. Totally understandable considering guard duty isn't the most consistent data structure in the world. However, I think Matano has a real opportunity to make a difference in the detecting engineering experience for malicious AWS activity by implementing support for guard duty finding as a managed AWS Log source, offering the same transformation into Elastic Common Schema and realtime detection benefits of Matano for guardduty.

As an user, this is roughly the experience that I'd hope to have with this manage log source.

1. Create a log source in the matano directory with the following contents
```yaml
# matano/log_sources/aws_guardduty/log_source.yml
name: "aws_guardduty"

managed:
  type: "AWS_GUARDDUTY"
```
2. Execute `matano deploy` and matano sets up the guardduty [Findings Export](https://docs.aws.amazon.com/guardduty/latest/ug/guardduty_exportfindings.html) infrastructure as well as required tables as seen in the other managed log sources
3. Navigate to athena and have the ability to query guardduty events that are transformed into elastic common schema as well as the ability to develop realtime detections on these guard duty events (same as the other managed sources). 

See [here for reference of what a guard duty event would look like transformed into ECS](https://www.elastic.co/docs/current/integrations/aws/guardduty#logs).
